### DISCUSSION TITLE
Generate one firmware image instead of 2 side specific image files.

### DISCUSSION THREAD

**Discussion by abreinig** [KinesisCorporation/Adv360-Pro-ZMK#508] [Ideas]
Why not copy what the glove80 does and generate one firmware file that you upload to both sides of the board and the differences are sorted out behind the scenes? This would prevent any user error of flashing a side with the wrong file and it would be extremely convenient.

**Comment by ReFil** [comment]
I'm going to move this topic over to the "Discussions" area

**Comment by foxnaren** [comment]
From my experience, just flashing the right side is enough. The left gets the file automatically.
